plugins/node/opentelemetry-instrumentation-knex/package.json

```diff
@@ -49,7 +49,8 @@
     "@opentelemetry/sdk-trace-node": "^1.8.0",
     "@types/mocha": "7.0.2",
     "@types/node": "18.6.5",
-    "knex": "0.95.9",
+    "better-sqlite3": "11.0.0",
+    "knex": "3.1.0",
```

Comment:
Note that knex@3 *does* drop support for node 14. Currently the Node.js v14 unit-tests passed, however.
I am more inclined to be testing with the recent knex than locking at the old version.  There may come a knex 3.x version that breaks our Node.js v14 tests.